sympy/functions/elementary/integers.py

```diff
@@ -153,13 +156,14 @@ def __gt__(self, other):
 class ceiling(RoundFunction):
     """
     Ceiling is a univariate function which returns the smallest integer
-    value not less than its argument. Ceiling function is generalized
-    in this implementation to complex numbers.
+    value not less than its argument. This implementation
+    generalizes ceiling to complex numbers as follows:
+        ceiling(a + b*I) -> ceiling(a) + ceiling(b)*I
```

Comment:
This line raised "unexpected indentation error" on Travis. I think bin/doctest may have caught it locally, but am not sure. The analog in floor probably has the same issue. 

Maybe use = (following math notation) or == (following Python notation), instead of -> ? I don't know if -> has a specific use in either context that fits here.